docs/install/tutorials/rpi.rst

```diff
@@ -426,18 +433,21 @@ After replicating your SD card and external storage device, you need re-register
+How many clients are supported?
+-------------------------------
 
-This work will be released in a future package targeting a pre-configured UWSGI and Nginx implementation, which will be possible to add by replacing the package of this tutorial with the new package. We will add a release note and update this tutorial accordingly.
+.. tip:: We recommend that you do your own benchmarking and share experiences in our `Community Forums <https://community.learningequality.org/>`_. See especially this thread about `choosing a MicroSD card <https://community.learningequality.org/t/microsd-cards-picking-the-right-one-experiences-and-benchmarks/935>`_.
 
-You may also want to install other services such as `Kiwix <https://www.kiwix.org>`_. If you have followed this tutorial, you can install Kiwix alongside Kolibri by downloading the ``kiwix-serve`` package and adding an Nginx configuration similar to the one we added for Kolibri.
+One of the bigger questions that may be hard to answer is how many students that can use Kolibri on a Raspberry Pi at the same time. One of the biggest bottle necks will be the Raspberry Pi's built-in Wi-Fi. Maybe it will support 5-10 devices that are located in a close perimeter.
 
 The Wi-Fi antenna and chip in the Raspberry Pi do not have capacity for many clients. Thus, you may also want to connect a stronger Access Point. If you intend to do this, you should modify the DHCP server (dnsmasq) to listen to the ``eth0`` device instead of ``wlan0``, switching off the Wi-Fi by removing ``hostapd``.
 
-There are several bottle necks in this setup, but we recommend that you focus on the strong sides of the Raspberry Pi platform: It's low-cost and uses little electricity. Perhaps you can connect it to solar power? Perhaps you can implement a good system for distributing software updates and replacement parts?
+If you are running behind an access point, a Raspberry Pi with 1 GB may support 10 clients, but your need to consider newer Raspberry Pi 4 devices if you want to support more than 10 clients.
+
+Future steps
+------------
+
+You may also want to install other services such as `Kiwix <https://www.kiwix.org>`_. If you have followed this tutorial, you can install Kiwix alongside Kolibri by downloading the ``kiwix-serve`` package and adding an Nginx configuration similar to :ref:`the one we added for Kolibri <nginx_custom_domain>`.
 
-.. tip:: Using a Raspberry Pi is subject to many tips and tricks not described here - please share your performance experiences in the `Community Forums <https://community.learningequality.org/t/microsd-cards-picking-the-right-one-experiences-and-benchmarks/935>`__.
+Lower capacity hardware like a Raspberry Pi poses a bottle neck, but we recommend that you focus on the strong sides of the platform: It's low-cost and uses little electricity. Perhaps you can connect it to solar power? Perhaps you can implement a good system for distributing software updates and replacement parts?
```

Comment:
The Raspberry Pi may not support many clients, but it is affordable and low-power.

Reply:
:+1: fixed in f755a99